docs/rfcs/20220410-rfc-process.md

```diff
@@ -0,0 +1,55 @@
+# RFC: The RFC (Request for Comments) Process
+
+* Author(s): [Ye Kuang](https://github.com/k-ye)
+* Date: 2022-04-10
+* Relevant Issue: N/A
+
+---
+
+# Summary
+
+This doc proposes the standard way to make design decisions in Taichi. RFCs should be proposed and discussed before any major design or implementation gets landed in Taichi.
+
+# Background
+
+Throughout the development of Taichi, we often hear new developers complaining about the steep learning curve of Taichi's codebase. One reason is that, currently the design decisions are spread across different places and hard to track.
+
+RFC process is a well-established mechanism in the open source communities to solve this problem. It provides a common place for developers to discuss design decisions publicly. It also serves as a doc library for people who want to learn about Taichi's architecture, evolving history, etc.
+
+# Goals
+
+* Provides a way for Taichi developers to discuss and to reach a consensus on any major changes openly and efficiently.
+* Serves as Taichi's public technical documentations.
+
+# Detailed Design
+
+1. Make a copy of [`yyyymmdd-rfc-template.md`](yyyymmdd-rfc-template.md), fill in all the required sections.
+2. Send a new PR with this RFC file, then go through the normal PR process.
+3. If the relevant community members decide to approve this RFC:
```

Comment:
How can the author(s) determine who are the relevant members of the RFC here, when it comes to the case that the author(s) aren't internal devs?

Reply:
I think a public Committer list should be fine for now.